BCStudentSoftwareDevTeam/celts
# Allow admins to send email through the event list page

**Summary:** 
Added functionality to the email buttons in the event list page (now the home page). Created a modal that pops up when the email button is clicked. The user can fill out the subject and body of the email as well as who is sending the email and who receives it. The senders can be the CELTS Admin or CELTS Student Staff. The recipients can be people who are interested in the program or only people who are RSVP'ed. 

**To Test:** 
Go to the website's main page and open one of the accordions (make sure the user is an Admin or CeltsStudentStaff). Click on the email button next to the email. Fill out the different fields and hit send.

**Files Changed**: 
default.yml
production.yml
staging.yml
routes.py
event.py
event_list.js
event_list.html
test_events.py
**Files Added:**
test_emailHandler.py
emailModal.html
emailHandler.py
email.py
